src/solver/type_getters.jl

```diff
@@ -20,11 +20,17 @@ function get_atmos(config::AtmosConfig, params)
     moisture_model = get_moisture_model(parsed_args)
     precip_model = get_precipitation_model(parsed_args)
     cloud_model = get_cloud_model(parsed_args)
+    ozone = get_ozone(parsed_args)
     radiation_mode = get_radiation_mode(parsed_args, FT)
     forcing_type = get_forcing_type(parsed_args)
     call_cloud_diagnostics_per_stage =
         get_call_cloud_diagnostics_per_stage(parsed_args)
 
+    if isnothing(ozone) && radiation_mode isa RRTMGPI.AbstractRRTMGPMode
+        @warn "prescribe_ozone is set to nothing with an RRTMGP model. Resetting to IdealizedOzone. This behavior will stop being supported in some future release"
```

Comment:
prescribe_ozone is default to false in default configs. Does this mean we want the user to specify true or false in the config? Why is it not a good idea to default it to IdealizedOzone if prescribe_ozone is not set (is this what you mean by prescribe_ozone is set to nothing)?

Reply:
Discussed offline: in a nutshell, we shouldn't be adding stuff by default (this is also our pattern for most other types)